rebasehelper/application.py

```diff
@@ -457,6 +458,12 @@ def _prepare_rebased_repository(self):
         if os.path.isfile(gitignore):
             shutil.copy(gitignore, self.rebased_sources_dir)
 
+        for source, _, source_type in self.spec_file.spc.sources:
+            if not urllib.parse.urlparse(source).scheme and source_type == 1:
```

Comment:
A comment would be nice. Will you remember what source type `1` means in a few months? :wink: